nsdb-sql/src/main/scala/io/radicalbit/nsdb/sql/parser/SQLStatementParser.scala

```diff
@@ -101,7 +101,8 @@ final class SQLStatementParser extends RegexParsers with PackratParsers {
   private val numbers          = """([0-9]+)""".r
   private val intValue         = numbers ^^ { _.toInt }
   private val longValue        = numbers ^^ { _.toLong }
-  private val floatValue       = """([0-9]+)\.([0-9]+)""".r ^^ { _.toDouble }
+  //TODO evaluate to change this name into something more consistent to its actual type
+  private val floatValue = """([0-9]+)\.([0-9]+)""".r ^^ { _.toDouble }
```

Comment:
can we effectively change this variable name into `doubleValue` as suggested by the TODO annotation